noir-projects/aztec-nr/aztec/src/context/interface.nr

```diff
@@ -239,18 +270,48 @@ struct AvmVoidCallInterface {
 }
 
 impl AvmVoidCallInterface {
-    pub fn call<N>(self, context: &mut AvmContext, gas_opts: GasOpts) {
+    pub fn call<N>(self, context: &mut AvmContext) {
+        let returns = context.call_public_function(
+            self.target_contract,
+            self.selector,
+            self.args,
+            GasOpts::default()
+        );
+        returns.assert_empty()
+    }
+
+    pub fn call_with_gas<N>(self, context: &mut AvmContext, gas_opts: GasOpts) {
```

Comment:
Should the call interface just always include gas? Or should we have a separate function like this with gas? (if so we probably want it for static/delegate too)